src/AwsClient.php

```diff
@@ -43,4 +44,9 @@ public function getSqsClient(array $extraOptions = [])
     {
         return new SqsClient(array_merge($this->getOptions(), $extraOptions));
     }
+
+    public function getSnsClient()
+    {
+        return new SnsClient($this->getOptions());
+    }
```

Comment:
In order to have this more adjustable please use:
In this way we can use this client on dev an overwrite the config.
```php
public function getSnsClient(array $extraOptions = [])
    {
        return new SnsClient(array_merge($this->getOptions(), $extraOptions));
    }
```